.pylintrc

```diff
@@ -556,7 +556,7 @@ contextmanager-decorators=contextlib.contextmanager
 # system, and so shouldn't trigger E1101 when accessed. Python regular
 # expressions are accepted.
 generated-members=
-
+    cv2.*
```

Comment:
Can you tell me why this has been added? Are we accessing non-existent members of cv2?

Reply:
This was added because cv2 functions are generated at runtime